cucumber/authenticators_status/features/authn_status.feature

```diff
@@ -100,7 +45,6 @@ Feature: Authenticator status check
       member: !user alice
     """
     And I am the super-user
-    And I successfully set OIDC variables
```

Comment:
if we dont need it can we remove the implementation as well?

Reply:
But we need the implementation in the OIDC tests. I only removed it from here so there is no dependency on OIDC as this test checks all the non-specific authenticator checks.